test/common/json/json_sanitizer_test_util.h

```diff
@@ -23,6 +23,32 @@ bool utf8Equivalent(absl::string_view a, absl::string_view b, std::string& errms
     EXPECT_TRUE(TestUtil::utf8Equivalent(a, b, errmsg)) << context << "\n" << errmsg;              \
   }
 
+/**
+ * Determines whether `sanitized`is equivalent to `original`. `original` may be
+ * in any encoding, e.g. ascii, binary, utf-8, gb2132. `sanitized` is valid JSON
+ * with `\u` escapes for any characters are not allowed in JSON strings, per
+ * https://www.json.org/json-en.html. We want to make sure our json encoding for
+ * `original` would be decoded into the same bytes.
+ *
+ * Note that the 'sanitized' argument does not accept arbitrary json string
```

Comment:
```suggestion
 * Note that the `sanitized` argument does not accept arbitrary json string
```

Reply:
And the @jmarantz

(I personally think ' is fine. But other positions in the comment have used `)